React/Views/RCTScrollView.m

```diff
@@ -317,8 +317,8 @@ - (void)setFrame:(CGRect)frame
   CGSize boundsSize = self.bounds.size;
 
   self.contentOffset = CGPointMake(
-    MAX(0, MIN(originalOffset.x, fullContentSize.width - boundsSize.width)),
-    MAX(0, MIN(originalOffset.y, fullContentSize.height - boundsSize.height)));
+    MAX(-contentInset.left, MIN(originalOffset.x, fullContentSize.width - boundsSize.width)),
```

Comment:
Sorry, I was terribly wrong. We have to fix this asap.

https://www.objc.io/issues/3-views/scroll-view/
> The property contentInset can change the maximum and minimum values of the content offset to allow scrolling outside of the scrollable area. Its type is UIEdgeInsets, which consists of 4 numbers: {top, left, bottom, right}. When you introduce an inset, you change the range of the content offset. For example, setting the content inset to have a value of 10 for its top value allows the content offset’s y value to reach -10. This introduces padding around the scrollable area.